build.gradle.kts

```diff
@@ -25,12 +25,14 @@ configurations {
 
 repositories {
 	mavenCentral()
+	maven { url = uri("https://packages.confluent.io/maven/") }
```

Comment:
Yes, because this tells Gradle to look for dependencies in Maven Central and also in the Confluent repository
Without it, I have exception:
`> Could not find io.confluent:kafka-protobuf-serializer:7.4.0`